shotover-proxy/src/transforms/cassandra/sink_cluster/routing_key.rs

```diff
@@ -54,7 +56,7 @@ fn serialize_routing_key_with_indexes(
 }
 
 // https://github.com/apache/cassandra/blob/3a950b45c321e051a9744721408760c568c05617/src/java/org/apache/cassandra/db/marshal/CompositeType.java#L39
```

Comment:
The linked source mentions the final byte being one of -1, 0 or 1, but we just hardcode to 0.
I dont really understand whats going on here but do we need to use any of those other values in certain cases?

Reply:
https://github.com/scylladb/scylladb/blob/40adf38915b6d8f5314c621a94d694d172360833/compound_compat.hh#L46

https://github.dev/datastax/java-driver/blob/a650ee43712cc0448ce6d263db30f15eb2f55c2c/core/src/main/java/com/datastax/oss/driver/internal/core/util/RoutingKey.java#L36

Some examples of other drivers doing the same.

Reply:
The first one is scylladb itself, not a driver, right?

If the java driver just hardcodes to 0, then it must be something fancy the server supports but the drivers can get away with ignoring.
Thanks for investigating :+1: